### PR TITLE
rpki: Add command to get detailed information

### PR DESCRIPTION
**Pull request by fujimoto-s**
This PR adds the command to get detailed information about RPKI validation.
(This implements #574.)

Example:
```bash
$ gobgp rpki detail 2.1.0.0/16 65001
Target Prefix: 2.1.0.0/16, AS: 65001
  This route is invalid  reason: as
  No VRP ASN matches the route origin ASN.

  Matched VRPs:
    No Entry
  Unmatched AS VRPs:
    Network            AS     MaxLen
    2.0.0.0/12         3215   16
    2.1.0.0/16         3215   16
  Unmatched Length VRPs:
    No Entry
```

### CODE REVIEW

**Comment by fujita** [comment]
Can we integrate this new API into the existing ValidateRib API?

**Comment by fujimoto-s** [comment]
So, the syntax should be like `gobgp rpki validate 2.1.0.0/16 3215`?
I think it is Yes, I'll fix it soon.

**Comment by fujita** [comment]
On the second thought, might be better to integrate into rib command. Junos does such.

https://www.juniper.net/documentation/en_US/junos12.2/topics/topic-map/bgp-origin-as-validation.html

**Comment by fujimoto-s** [comment]
Sorry, I could not find that Junos outputs a detailed information about RPKI in the page.

But, I think it is good to integrate it into RIB command. 
So, `gobgp global rib validation 2.1.0.0/16 3215` is good?
If so, I will implement as such.

**Comment by fujimoto-s** [comment]
I will fix this command like below:
`gobgp neighbor 125.0.0.1 local validation 2.1.0.0/16`

* use `neighbor` command to show detailed information
* not specify origin_as
  * respond with all routes which match to the specified prefix

**Comment by fujita** [comment]
gobgp neighbor 125.0.0.1 local validation 2.1.0.0/16

this can be used with only route server use-case.

gobgp neighbor 125.0.0.1 adj-in validation 2.1.0.0/16

?

**Comment by fujimoto-s** [comment]
Sorry, I did not intended to limit it to `local` only.

More precisely, the command syntax will be like below:
`gobgp neighbor <neighbor-address> {adj-in|local|adj-out} validaton <prefix>`
Does it match to your image?

**Comment by fujita** [comment]
Let's start with 'adj-in'. If someone requests the rests (I doubt), let's implement them.

**Comment by fujimoto-s** [comment]
I see, I will do that.

**Comment by fujita** [comment]
I suspect that the CI failed due to logrus package name breakage. Can you rebase against the current master and run the CI again?

**Comment by fujimoto-s** [comment]
Thanks, I rebased & pushed!

File: docs/sources/cli-command-syntax.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ The following options can be specified in the global subcommand:
 % gobgp neighbor <neighbor address> [local|adj-in|adj-out] [<prefix>|<host>] [longer-prefixes|shorter-prefixes] [-a <address family>]
 # show table summary
 % gobgp neighbor <neighbor address> [local|adj-in|adj-out] summary [-a <address family>]
+# show RPKI detailed information in adj-in table
+% gobgp neighbor >neighbor address> adj-in <prefix> validation
```
Comment (fujita) [inline docs/sources/cli-command-syntax.md]: >neighbor
typo?

Comment (fujimoto-s) [inline docs/sources/cli-command-syntax.md]: Sorry, this is typo.
I will fix it.

**Comment by fujita** [comment]
Thanks!

**Pull request merged**